.travis.yml

```diff
@@ -35,30 +35,28 @@ jobs:
       install:
         - composer install $COMPOSER_FLAGS
       script:
-        - make autoreview --keep-going
```

Comment:
As far as I can tell the `--keep-going` is not doing anything

Reply:
Yes. If there's a single failure, you may want to know if it is the only failure. Because the first failure may not point to the root of the problem.